src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public double getSide() {
+        return side;
+    }
+
+    public void setSide(double side) {
+        this.side = side;
+    }
+
+    @Override
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. If there is another method like `getArea()` that is conceptually different from `draw()`, it should be in a different interface.